docs/user/openstack/README.md

```diff
@@ -646,6 +646,51 @@ If you ran the installer with a [custom CA certificate](#self-signed-openstack-c
 oc edit configmap -n openshift-config cloud-provider-config
 ```
 
+### Enabling Octavia for Load Balancer Services
+
+OpenShift supports creating load balancer service types, and ingress controllers with load balancers as a backend using octavia as a day 2 operation. However, there are a few known issues to be aware of:
+* only TCP traffic is supported
+* the floating IPs attached to your Octavia load balancers will not be deleted if you delete your cluster
+* `manage-security-groups` will not work for non-admin tenants
+* `loadBalancerSourceRanges` is currently [not supported](https://bugzilla.redhat.com/show_bug.cgi?id=1980748)
+
+Enabling this service should only be done with non-kuryr deployments as a day 2 operation. To do this, edit the cloud provider config:
+
+```sh
+oc edit configmap -n openshift-config cloud-provider-config
+```
+
+If you are using the `amphora` driver, then you should add this to your cloud provider config:
+
+```
+[LoadBalancer]
+use-octavia = true
+lb-provider = amphora
+manage-security-groups = true
```

Comment:
Wait, why it's set to true? It won't work for non-admin tenants.

Reply:
fixed, my bad